jvm-packages/xgboost4j-gpu/src/native/jvm_utils.h

```diff
@@ -0,0 +1,17 @@
+#ifndef JVM_UTILS_H_
+#define JVM_UTILS_H_
+
+#include "./xgboost4j-gpu.h"
+
+#define JVM_CHECK_CALL(__expr)                                                 \
```

Comment:
I think we have an utility inside the original jvm wrapper header right?

Reply:
yeah, just this one.

Reply:
I meant we have the same code in xgboost4j/src/native right?